src/Library/helpers.php

```diff
@@ -1,17 +1,38 @@
 <?php
 
+use Illuminate\Support\Collection;
+use Illuminate\Pagination\LengthAwarePaginator;
 use OwowAgency\LaravelResources\Resources\ResourceFactory;
 
 if (! function_exists('resource')) {
     /**
      * Makes the resource for the specified model class.
      *
      * @param  mixed  $model
-     * @param  boolean  $isCollection
+     * @param  boolean  $isCollection|null
      * @return mixed
      */
-    function resource($model, $isCollection = false)
+    function resource($model, $isCollection = null)
```

Comment:
It feels like this whole implementation should be in the `ResourceFactory`. The helper function is to make our lives easier by not having to cann the `(new ResourceFactory())->make()`